hardhat.config.js

```diff
@@ -70,21 +70,21 @@ module.exports = {
         settings: {
           optimizer: {
             enabled: true,
-            runs: 200
-          }
-        }
+            runs: 200,
+          },
+        },
       },
       { version: "0.6.12" },
       { version: "0.5.17" },
```

Comment:
Btw, why do we need both 0.5.17 and 0.5.16, do we really need both Solidity versions here?

Reply:
I believe this is a mistake and as far as I remember it is fixed in the lbp PR.